client/src/components/DisplayTable/DisplayTableUtils.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../common_css/common-variables';
+
+.dp-utils-heading-container {
+  display: flex;
+  flex-direction: row-reverse;
```

Comment:
Row reverse is a little annoying, because the order in JSX will be reversed from the order on the live site. `justify-content: flex-end` will also push the util icons to the right side, and it won't reverse their order in the process.